packages/widgetbook_cli/bin/version.dart

```diff
@@ -1,3 +1 @@
-import 'package:version/version.dart';
-
-Version currentVersion = Version(0, 1, 0);
+const String currentVersion = '0.1.0-dev.35';
```

Comment:
This is not the correct version which makes me doubt that the update command is properly implemented.

Reply:
I left this there deliberately to test if the update works.

Reply:
Okay. But in the final release this value has to be correct 👍🏻